client/packages/common/src/ui/components/inputs/TextInput/PasswordTextInput.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC, useState } from 'react';
+import { IconButton, StandardTextFieldProps } from '@mui/material';
+import { BasicTextInput } from './BasicTextInput';
+import { EyeIcon, EyeOffIcon } from '@common/icons';
+import { useTranslation } from '@common/intl';
+
+export type PasswordTextInputProps = StandardTextFieldProps & {
+  fixedHeight?: boolean;
+};
+
+export const PasswordTextInput: FC<PasswordTextInputProps> = React.forwardRef(
+  (props, ref) => {
+    // if the helper text is a space then the height of the component doesn't change
+    // when the helper text is shown / removed
+    const defaultWarning = props.fixedHeight ? ' ' : '';
+    const [showPassword, setShowPassword] = useState(false);
+    const [warning, setWarning] = useState(defaultWarning);
+    const t = useTranslation();
+    const visibilityInputButton = (
+      <IconButton
+        aria-label="toggle password visibility"
```

Comment:
Can I suggest an addition?
```
       disabled={props.disabled}
```

when on the initialisation screen, after clicking initialise, the password input is disabled but you can still click the magic eye..

Reply:
🙏 de1926edebbe71c78929649d74cb972c986ae03b and I tested